src/name/mlopatkin/andlogview/Main.java

```diff
@@ -60,6 +61,20 @@ public static void main(String[] args) {
             commandLine = new CommandLine();
         }
 
+        if (SystemUtils.IS_OS_MACOS) {
+            // Move JMenuBar to macOS native global Menu bar.
+            System.setProperty("apple.laf.useScreenMenuBar", "true");
+            // Change App name in menu bar.
+            System.setProperty("apple.awt.application.name", "AndLogView");
+            // Force default light style even with global system dark mode to fix black-on-black text in some controls.
+            System.setProperty("apple.awt.application.appearance", "NSAppearanceNameAqua");
+            try {
+                UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
```

Comment:
Is it safe to set up LaF outside of the EDT?

Reply:
As per docs https://docs.oracle.com/javase/tutorial/uiswing/lookandfeel/plaf.html
> Note: If you are going to set the L&F, you should do it as the very first step in your application. Otherwise you run the risk of initializing the Java L&F regardless of what L&F you've requested. 

Btw, the problems described in the comments can be resolved by building on and bundling newer JDK/JRE (like 16/17) or by using some custom LaF like [VAqua](https://violetlib.org/vaqua/overview.html). This one looks pretty decent. [Flatlaf](https://www.formdev.com/flatlaf/) is also good for cross-platform usage. They both should support global dark mode.

Reply:
Ok, thanks, sgtm there. I guess flatlaf is the future, I'm considering switching to it eventually. Metal has the advantage of being available out of the box, everywhere.